.github/workflows/ci.yml

```diff
@@ -0,0 +1,69 @@
+name: CI
+
+on: [push, pull_request]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    strategy:
+      matrix:
+        php: [7.1, 7.2, 7.3, 7.4, 8.0]
+        lib:
+          - { laravel: ^8.0, testbench: ^6.0, composer_flags: --prefer-stable }
+          - { laravel: ^7.0, testbench: ^5.0, composer_flags: --prefer-stable }
+          - { laravel: ^6.0, testbench: ^4.0, composer_flags: --prefer-stable }
+          - { laravel: ^6.0, testbench: ^4.0, composer_flags: --prefer-stable --prefer-lowest }
+          - { laravel: 5.8.*, testbench: 3.8.*, composer_flags: --prefer-stable }
+          - { laravel: 5.7.*, testbench: 3.7.*, composer_flags: --prefer-stable }
+          - { laravel: 5.6.*, testbench: 3.6.*, composer_flags: --prefer-stable }
+        exclude:
+          - lib: { laravel: ^8.0, testbench: ^6.0, composer_flags: --prefer-stable }
+            php: 7.2
+          - lib: { laravel: ^8.0, testbench: ^6.0, composer_flags: --prefer-stable }
+            php: 7.1
+          - lib: { laravel: ^7.0, testbench: ^5.0, composer_flags: --prefer-stable }
+            php: 7.1
+          - lib: { laravel: ^6.0, testbench: ^4.0, composer_flags: --prefer-stable }
+            php: 7.1
+          - lib: { laravel: ^6.0, testbench: ^4.0, composer_flags: --prefer-stable --prefer-lowest }
+            php: 7.1
+          - lib: { laravel: 5.8.*, testbench: 3.8.*, composer_flags: --prefer-stable }
+            php: 8.0
+          - lib: { laravel: 5.7.*, testbench: 3.7.*, composer_flags: --prefer-stable }
+            php: 8.0
+          - lib: { laravel: 5.6.*, testbench: 3.6.*, composer_flags: --prefer-stable }
+            php: 8.0
+
+
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: Setup PHP
+        uses: shivammathur/setup-php@v2
+        with:
+          php-version: ${{ matrix.php }}
+          coverage: xdebug
+
+      - name: Adjust package versions
+        run: |
+          composer require "laravel/framework:${{ matrix.lib.laravel }}" --dev --no-update ${{ matrix.lib.composer_flags }}
+          if php -r 'exit((int)!version_compare(PHP_VERSION, "8", ">="));'; then
+            composer require "orchestra/testbench:>=4.14" --dev --no-update ${{ matrix.lib.composer_flags }}
```

Comment:
The lowest version of `testbench` supporting PHP 8 is `4.9.0` but `4.14.0` is needed considering `mockery`, its dependency.